activerecord/lib/active_record/tasks/sqlite_database_tasks.rb

```diff
@@ -9,9 +9,8 @@ def self.using_database_configurations?
         true
       end
 
-      def initialize(db_config, root = ActiveRecord::Tasks::DatabaseTasks.root)
+      def initialize(db_config)
         @db_config = db_config
-        @root = root
```

Comment:
not really related to this PR, but this var isn't used for anything so i removed it.